src/TeamPasswordManagerClient/TpmPasswordClient.cs

```diff
@@ -117,6 +125,14 @@ public async Task<PasswordDetails> GetPassword(int id)
             return Json.ToObject<PasswordDetails>(response);
         }
 
+        public async Task<IEnumerable<PasswordEntry>> FindPassword(string passwordName)
+        {
+            var urlencodedPassword = HttpUtility.UrlEncode($"\"{passwordName}\"");
```

Comment:
Actually I think Uri.EscapeDataString is better - perhaps this is the bug in the SearchPassword func?